sdk/translation/azure-ai-translation-text/README.md

```diff
@@ -63,8 +63,44 @@ With the value of the `endpoint`, `credential` and a `region`, you can create th
 <!-- SNIPPET: sample_text_translation_client.create_text_translation_client_with_credential -->
 
 ```python
+from azure.ai.translation.text import TextTranslationClient, TranslatorCredential
+
+endpoint = os.environ["AZURE_TEXT_TRANSLATION_ENDPOINT"]
+apikey = os.environ["AZURE_TEXT_TRANSLATION_APIKEY"]
+region = os.environ["AZURE_TEXT_TRANSLATION_REGION"]
+
 credential = TranslatorCredential(apikey, region)
-text_translator = TextTranslationClient(credential, endpoint)
+text_translator = TextTranslationClient(endpoint=endpoint, credential=credential)
+```
+
+<!-- END SNIPPET -->
+
+#### Create a `TextTranslationClient` using Entra (Azure Active Directory) Authentication
+
+`AzureKeyCredential` authentication is used in most of the examples, but you can also authenticate with Entra (Azure Active Directory) using the [Azure Identity library][azure_identity].  Note that regional endpoints do not support AAD authentication.
```

Comment:
Note that regional endpoints do not support AAD authentication.

I am not sure that this is the naming we are using in translator documentation, can you please provide the link to translator API docs, where we are calling our global endpoint "regional endpoint"?

Reply:
That text was taking directly from the Document Translation README.md file.  Changed to "global endpoint".